src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static boolean isNonAddTaskInput (String input, List<Task> tasks) {
+        if (input.contains("done")) {
+            int taskIndex = Integer.parseInt(input.replaceAll("[^0-9]", ""));
+            tasks.get(taskIndex - 1).markAsDone();
+            Task.numberOfTasks -= 1;
+            return true;
+        }
+
```

Comment:
I like how neat this is but according to the coding standard it should have this form:
if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
    statements;
}